code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm

```diff
@@ -106,6 +114,7 @@
 	name = "Tricordrazine"
 	id = "tricordrazine"
 	description = "Tricordrazine is a highly potent stimulant, originally derived from cordrazine. Can be used to treat a wide range of injuries."
+	taste_description = "grossness"
```

Comment:
If you go through the trouble of getting a glass to drink an injectable or pill-able medicine, I'm just gonna tilt my head at you, okay?